src/root.component.tsx

```diff
@@ -25,7 +25,7 @@ const Root: React.FC = () => {
         <Routes>
           <Route path="/" element={<FacilityMetrics />} />
           <Route path="/fhir-exchange" element={<Fhir />} />
-          <Route path="/sync-task-types" element={<SyncTaskTypes />} />
+          <Route path="/schedule-manager" element={<ScheduleManager />} />
```

Comment:
looks like u just removed your route to sync-task-types yet you needed to add it below schedule-manager